runtime/lua/vim/lsp/_completion.lua

```diff
@@ -165,34 +204,18 @@ function M.omnifunc(findstart, base)
     local params = util.make_position_params(win, client.offset_encoding)
     client.request(ms.textDocument_completion, params, function(err, result)
       if err then
-        log.warn(err.message)
+        require('vim.lsp.log').warn(err.message)
```

Comment:
Why do you re-importing `log` here?

Reply:
Given that this is the only use I removed the other import and placed it here to make it lazy